HS9_py2/heatsource9/Stream/PyHeatsource.pyx

```diff
@@ -163,13 +163,15 @@ def CalcSolarPosition(lat, lon, hour, min, sec, offset,
         else:
             Azimuth_mod = Azimuth
         tran = bisect(AngleStart,Azimuth_mod)-1
+
     return Altitude, Zenith, Daytime, tran, Azimuth_mod
 
 def GetStreamGeometry(Q_est, W_b, z, n, S, D_est, dx, dt):
-    Converge = 10
-    dy = 0.01
-    count = 0
-    power = 2/3
+    cdef double Converge = 10

```

Comment:
Defining all variables as C types increases ability for the compiler to optimize.